source/documentation/continuous-integration-and-delivery/services.md

```diff
@@ -0,0 +1,14 @@
+## Services
+
+For convenience, the Reliability Engineering team provision the following
+resources for each Concourse team:
+
+- A private AWS S3 bucket (publicly readable from the internet by anyone)
```

Comment:
I'm still a little confused on this one. Could you just clarify what is 'private' about this bucket given it's publicly readable from the internet by anyone? And the difference between this bucket and the 'public' mentioned in the line below? Cheers

Reply:
I am not good at computer :bowtie: , please see 85ea494